src/components/Navbar/navbar.css

```diff
@@ -0,0 +1,135 @@
+.navbarContainer {
+  width: 100%;
+  background-color: crimson;
+  color: white;
+}
+.navbarWrapper {
+  width: 93.75%;
+  height: 80px;
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+}
+.navbarLeft {
+  color: white;
+  padding-left: 10px;
+  display: flex;
+  align-items: center;
+  z-index: 1000;
+}
+
+.logo {
+  width: 32px;
+  height: 32px;
+  color: white;
+  padding-right: 10px;
+  border-radius: 50%;
+  object-fit: cover;
+}
+.logo-name {
+  font-size: 18px;
+  font-weight: 600;
+}
+.navbarRight {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.hideNavbarMenu {
+  padding: 0;
+  display: flex;
+  list-style: none;
+  align-items: center;
+}
+
+@media (max-width: 768px) {
+  .hideNavbarMenu {
+    top: 0;
+    background-color: #524141;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    list-style: none;
+    position: fixed;
+    right: -100%;
+    width: 100%;
+    height: 100vh;
+    z-index: 100;
+    transition: all 0.5s linear;
+  }
+}
+
+.navbarMenu {
+  padding-right: 20px;
+  font-size: 20px;
+}
+@media (max-width: 768px) {
```

Comment:
You have two same declarations for @media in a single file. I believe you are not using the style component right?
You may want to take the navbarMenu @media CSS to the upper @media.